contrib/haproxy/files/sysusers.conf

```diff
@@ -0,0 +1,3 @@
+# create haproxy user 
+
+u _haproxy - "haproxy user" /var/empty /usr/bin/nologin
```

Comment:
since introduction of sd-sysusers, it should no longer be necessary to explicitly specify nologin path (it's the same as the default now)